imports/plugins/core/email/server/no-meteor/register.js

```diff
@@ -0,0 +1,62 @@
+import mutations from "./mutations";
```

Comment:
I feel like moving these to `no-meteor/register` is confusing. Does moving it to this directory enable any different functionality or is it just a mnemonic?

Reply:
It's mnemonic, but the new way of exporting a single `register` function that accepts `app` argument is important. It's a stepping stone toward NPM packages. Imagine the next step where everything in `no-meteor` moves to a separate package. Then the original, outer `register.js` file will `import register from "@reactioncommerce/plugin-email/register";` and pass it the `app` to connect it.

The step after that is removing the Meteor auto-register stuff and having people explicitly call the plugins in the order they wish, like is done in `registerPlugins.js`. That ultimately will be the pattern when the API is fully transitioned to pure Node app.